firmware/lm32/ad9984a.c

```diff
@@ -0,0 +1,94 @@
+#include <stdio.h>
```

Comment:
Can you add some comments linking to were datasheets for this part can be found.

It would also be good to add some comments about the VGA board here.